server/src/test/java/org/candlepin/util/DateRangeTest.java

```diff
@@ -35,33 +36,27 @@
     public void getters() {
         Date start = TestDateUtil.date(2012, 5, 22);
         Date end = TestDateUtil.date(2012, 7, 4);
-        DateRange range = new DateRange(start, end);
-
-        assertEquals(start, range.getStartDate());
-        assertEquals(end, range.getEndDate());
+        DateRange range = new DateRange();
+        range.startDate(Util.toDateTime(start)).endDate(Util.toDateTime(end));
+        assertEquals(start, Util.toDate(range.getStartDate()));
+        assertEquals(end, Util.toDate(range.getEndDate()));
     }
 
     @Test
     public void contains() {
-        DateRange range = new DateRange(TestDateUtil.date(2001, 7, 5), TestDateUtil.date(2010, 7, 4));
-
-        assertTrue(range.contains(TestDateUtil.date(2005, 6, 9)));
-        assertFalse(range.contains(TestDateUtil.date(1971, 7, 19)));
-        assertFalse(range.contains(TestDateUtil.date(2012, 4, 19)));
-        assertTrue(range.contains(TestDateUtil.date(2001, 7, 5)));
-        assertTrue(range.contains(TestDateUtil.date(2010, 7, 4)));
+        Date start = TestDateUtil.date(2001, 7, 5);
```

Comment:
We should drop this whole test class. `DateRange.contains` was no longer in use and there is no need to test getters of generated class.

Reply:
Done